src-srv/utils/transformations/newsdoc/tt/visual.ts

```diff
@@ -7,34 +7,38 @@ export const transformVisual = (element: Block): TBElement => {
   return {
     id,
     class: 'block',
-    type: 'core/image',
+    type: 'tt/visual',
     properties: {
-      // FIXME: This is a hack to get a viewable image
-      // TODO: Could there be more than one link/image?
-      src: links[0].url?.replace('_NormalPreview.jpg', '_WatermarkPreview.jpg') ?? '',
+      // FIXME: This is a hack to get a viewable image,
+      // we need authentication to view non-watermarked images
+      href: links[0]?.url
+        ?.replace('_NormalPreview.jpg', '_WatermarkPreview.jpg') ?? '',
       rel: links[0].rel,
       uri: links[0].uri,
       type: links[0].type,
       credit: links[0].data.credit,
-      altText: data.caption,
+      altText: data?.altText,
```

Comment:
Should it be set to undefined rather than empty string?

Reply:
After consideration, all values should be defined as their unset representation in Go. Ie string '', number 0 etc.
altText isn't a valid newdoc attribute so i've removed it. We should decide if we should have it.